packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -325,13 +325,15 @@ class CupertinoDynamicColor extends Color {
   /// Resolves the given [Color] by calling [resolveFrom].
   ///
   /// If the given color is already a concrete [Color], it will be returned as is.
+  /// If the given color is null, returns null for convenience.
```

Comment:
You don't really need to say "for convenience".

Reply:
Aye